automation_tools/__init__.py

```diff
@@ -2977,7 +2977,15 @@ def download_manifest(url=None, consumer=None):
     """
     user = os.environ.get('RHN_USERNAME')
     password = os.environ.get('RHN_PASSWORD')
-    base64string = base64.encodestring('%s:%s' % (user, password)).strip()
+    string = u'{0}:{1}'.format(user, password)
```

Comment:
If we are using `u'{0}:{1}` then the string would always be of `<type 'unicode'>`.

Reply:
```u'some string'```:
- python3 => str
- python2 => unicode

we need to avoid asking about 'unicode' since this type doesn't exist in python3